File: data/CodeLlama/readable_format_HPEC/fortran/file213.f90
subroutine print_array(n, w)
implicit none

DATA_TYPE, dimension(n) :: w
integer :: n
integer :: i, j
do i = 1, n
write(0, DATA_PRINTF_MODIFIER) w(i)
if (mod(i - 1, 20) == 0) then
write(0, *)
end if
end do
write(0, *)
end subroutine

File: data/gpt-4o/readable_format_HPEC/cpp/file213.cpp
#include <iostream>
#define DATA_TYPE int
#define DATA_PRINTF_MODIFIER "%d"

void print_array(int n, DATA_TYPE* w)
{
    for (int i = 0; i < n; i++)
    {
        printf(DATA_PRINTF_MODIFIER, w[i]);
        if ((i + 1) % 20 == 0)
            printf("\n");
    }
    printf("\n");
}
